sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -36,6 +38,16 @@ import com.mapbox.maps.renderer.egl.EGLCore
 open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
   private var mapController: MapController
 
+  private var isViewAnnotationManagerInitialized = false
+  /**
+   * Get view annotation manager instance to add / update / remove view annotations
+   * represented as Android views.
+   */
+  val viewAnnotationManager: ViewAnnotationManager by lazy {
```

Comment:
it's aligned with accessing other plugins although it's not a plugin. it's not a plugin because it uses core part of gl-native and even if user will disable that as a plugin - core part will be still in place. so it's logical to leave platform part also as part of SDK but separate functionality to it's own manager
e.g. `mapView.camera`, `mapView.logo`

Reply:
Lets talk, not sure I got the full context and API surface yet